frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/BaseRebaseBranchOntoParentAction.java

```diff
@@ -106,22 +112,34 @@ protected void doRebase(AnActionEvent anActionEvent, BaseGitMacheteNonRootBranch
     doRebase(project, macheteRepository, gitRepository, branchToRebase);
   }
 
-  private void doRebase(Project project, IGitMacheteRepository macheteRepository, GitRepository repository,
+  private void doRebase(Project project, IGitMacheteRepository macheteRepository, GitRepository gitRepository,
       BaseGitMacheteNonRootBranch branchToRebase) {
+    LOG.debug(() -> "Enter BaseRebaseBranchOntoParentAction#doRebase(project = ${project}, " +
+        "macheteRepository = ${macheteRepository}, gitRepository = ${gitRepository}, " +
+        "branchToRebase = ${branchToRebase} (${branchToRebase.getName()})");
     Try.of(() -> macheteRepository.getParametersForRebaseOntoParent(branchToRebase))
-        .onSuccess(gitRebaseParameters -> new Task.Backgroundable(project, "Rebasing") {
-          @Override
-          public void run(ProgressIndicator indicator) {
-            GitRebaseParams params = getIdeaRebaseParamsOf(repository, gitRebaseParameters);
-            GitRebaseUtils.rebase(project, List.of(repository), params, indicator);
-          }
-
-          // TODO (#95): on success, refresh only sync statuses (not the whole repository). Keep in mind potential
-          // changes to commits (eg. commits may get squashed so the graph structure changes).
-        }.queue()).onFailure(e -> {
+        .onSuccess(gitRebaseParameters -> {
+          LOG.debug(() -> "Queuing \"${branchToRebase.getName()}\" branch rebase background task");
+          new Task.Backgroundable(project, "Rebasing") {
+            @Override
+            public void run(ProgressIndicator indicator) {
+              GitRebaseParams params = getIdeaRebaseParamsOf(gitRepository, gitRebaseParameters);
+              LOG.info(() -> "Rebasing \"${gitRebaseParameters.getCurrentBranch().getName()}\" branch " +
+                  "until ${gitRebaseParameters.getForkPointCommit().getHash()} commit " +
+                  "onto ${gitRebaseParameters.getNewBaseCommit().getHash()}");
+              GitRebaseUtils.rebase(project, List.of(gitRepository), params, indicator);
+            }
+
+            // TODO (#95): on success, refresh only sync statuses (not the whole repository). Keep in mind potential
+            // changes to commits (eg. commits may get squashed so the graph structure changes).
+          }.queue();
+        }).onFailure(e -> {
           // TODO (#172): redirect the user to the manual fork-point
           var message = e.getMessage() == null ? "Unable to get rebase parameters." : e.getMessage();
+          LOG.error(message);
           VcsNotifier.getInstance(project).notifyError("Rebase failed", message);
+          GuiUtils.invokeLaterIfNeeded(() -> Messages.showErrorDialog(message, "Something Went Wrong..."),
+              ModalityState.NON_MODAL);
```

Comment:
Hmm possibly for the sake of better readability we should actually increase the maximum line size :thinking:

Reply:
@mkondratek WDYT?

Reply:
The present `lineSplit` is set to 120. I must say that I am used to this length. On the other hand, the situation when the line *almost* fits the line length seems to be very common to me and 120 is a weird number. What about some nice round and greater value, like 124? :thinking: I would not like to go any further because it might become a nuisance during file comparison where they are two (or in case of merging even :three: :exclamation: ) files next to each other.

Reply:
Heh, or 128 :)

Reply:
:man_facepalming:

Reply:
Ok, increased to 128 but this is only 8 chars more